x-pack/plugins/siem/public/components/page/network/users_table/columns.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { StaticIndexPattern } from 'ui/index_patterns';
+
+import { FlowTarget, UsersItem } from '../../../../graphql/types';
+import { networkModel } from '../../../../store';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { getEmptyTagValue } from '../../../empty_value';
+import { Columns } from '../../../load_more_table';
+import { Provider } from '../../../timeline/data_providers/provider';
+
+import * as i18n from './translations';
+import { getRowItemDraggables } from '../../../tables/helpers';
+
+export const getUsersColumns = (
+  indexPattern: StaticIndexPattern,
+  ip: string,
+  flowTarget: FlowTarget,
+  type: networkModel.NetworkType,
+  tableId: string
+): [
+  Columns<UsersItem['name']>,
+  Columns<UsersItem['id']>,
+  Columns<UsersItem['groupName']>,
+  Columns<UsersItem['groupId']>,
+  Columns<UsersItem['count']>
+] => [
+  {
+    field: `node.user.name`,
```

Comment:
You don't need interpolation. We probably should add a linter rule to not use interpolation when there is no variables.

Reply:
Thanks! And I can't seem to find an appropriate linter rules for this. There's just [prefer-template](https://eslint.org/docs/rules/prefer-template), [no-template-curly-in-string](https://eslint.org/docs/rules/no-template-curly-in-string) and [allowTaggedTemplates](https://eslint.org/docs/rules/no-unused-expressions#allowtaggedtemplates) (eslint only).